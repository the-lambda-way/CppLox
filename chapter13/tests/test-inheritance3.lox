class Doughnut {
  cook() {
    print "Fry until golden brown.";
  }
}

class BostonCream < Doughnut {
  cook() {
    super.cook();
    print "Pipe full of custard and coat with chocolate.";
  }
}

BostonCream().cook();
