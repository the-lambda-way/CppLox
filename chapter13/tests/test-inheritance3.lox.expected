Fry until golden brown.
Pipe full of custard and coat with chocolate.
